com.cubrid.cubridmanager.core/src/com/cubrid/cubridmanager/core/common/model/ConfConstants.java

```diff
@@ -332,6 +337,12 @@ public static String[][] getBrokerParameters(ServerInfo serverInfo) {
 		if (!isSupportEnableAccessControl) {
 			length = length - 2;
 		}
+		if (!isSupportEnableOpenSSL) {
+            length = length - 1;
+        }
```

Comment:
need some indentation

Reply:
There are many changes when applying indentation.
I'll apply the indentation after the review is done.